deploy/olm-catalog/kogito-cloud-operator/0.6.0/kogito-cloud-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -4,21 +4,6 @@ metadata:
   annotations:
     alm-examples: |-
       [
-        {
```

Comment:
I guess this is just a matter of formatting the yaml, but please just double check that your SDK is set to 0.11.0